apps/emqx_bridge_http/docker-ct

```diff
@@ -0,0 +1 @@
+toxiproxy
```

Comment:
The test suites don't seem to use toxiproxy, so I think this file could be removed.